internal/services/hdinsight/hdinsight_spark_cluster_resource_test.go

```diff
@@ -40,6 +40,26 @@ func TestAccHDInsightSparkCluster_basic(t *testing.T) {
 	})
 }
 
+func TestAccHDInsightSparkCluster_availabilityZones(t *testing.T) {
```

Comment:
If zones are update-able then this test should be extended to update it. Furthermore this property should be added to the `complete` test config as well.

Reply:
Added to complete test.

Reply:
@jiaweitao001 
> If zones are update-able then this test should be extended to update it

This comment hasn't been addressed, currently we're not testing the update of this property, can you please add test coverage for this?

Reply:
Hi @stephybun , this property is not updatable. I've added description in the docs.

Reply:
If this property isn't update-able then we should be using the ForceNew commonschema for zones so that the resource will be recreated if it's changed

`commonschema.ZonesMultipleOptionalForceNew()`